manifest.json

```diff
@@ -16,12 +16,20 @@
         {
             "matches": ["https://anilist.co/anime/*"],
             "js": ["anilist/content.js"]
-        }
+        },
+		{
+			"matches": ["https://kitsu.io/anime/*"],
+			"js": ["kitsu/content.js"]
+		}
     ],
-    "web_accessible_resources": ["myanimelist/script.js", "anilist/script.js"],
+    "background": {
```

Comment:
remove this line

Reply:
This is the same as above. We need a background script as well to make it work.